ui/src/fields/MultipleSelectField/MultipleSelectField.module.css

```diff
@@ -1,109 +1,13 @@
-/*            background-color: #c8e5ff;*/
-/*            color: @black;*/
-/*            margin-top: 6px;*/
-/*            margin-left: 6px;*/
-
-/*            &__remove {*/
-/*                color: rgba(0, 0, 0, 0.85);*/
-/*            }*/
-/*        }*/
-
-/*        .ant-select-search__field__wrap {*/
-/*            font-family: open_sansregular;*/
-/*            font-size: 16px;*/
-/*            color: rgba(0, 0, 0, 0.85);*/
-/*        }*/
-/*    }*/
-/*}*/
-
-/*.checkbox {*/
-/*    width: 16px;*/
-/*    height: 16px;*/
-/*    top: -0.13em;*/
-/*    position: relative;*/
-/*    display: inline-block;*/
-/*}*/
-
-/*.emptyCheckbox {*/
-/*    width: 16px;*/
-/*    height: 16px;*/
-/*    border-radius: 2px;*/
-/*    border: solid 1px @brown-grey;*/
-/*    top: 0.09em;*/
-/*    position: relative;*/
-/*    display: inline-block;*/
-/*}*/
-
-/*.title {*/
-/*    padding-left: 10px;*/
-/*}*/
-
-/*.dropDownMenu {*/
-/*    :global {*/
-/*        .ant-select-dropdown-menu-item {*/
-/*            font-family: open_sansregular;*/
-/*            font-size: 16px;*/
-/*            color: @black;*/
-
-/*            &:hover:not(ant-select-dropdown-menu-item-disabled) {*/
-/*                color: #176db7;*/
-/*                background-color: rgba(230, 230, 230, 0.3);*/
-/*                opacity: 1;*/
-/*            }*/
-/*        }*/
-
-/*        .ant-select-dropdown-menu-item-selected {*/
-/*            color: #176db7;*/
-/*            opacity: 1;*/
-/*            font-weight: 400;*/
-/*            background-color: #ffffff;*/
-/*        }*/
-
-/*        .ant-select-dropdown-menu-item-active:not(.ant-select-dropdown-menu-item-disabled) {*/
-/*            background-color: #ffffff;*/
-/*        }*/
-/*    }*/
-/*}*/
-
-/*.container:global(.ant-select-focused.ant-select-enabled .ant-select-selection) {*/
-/*    border-color: #176db7;*/
-/*    box-shadow: none;*/
-/*}*/
-
-/*:global(.ant-select:not(.ant-select-disabled)).container {*/
-/*    :global {*/
-/*        .ant-select-selection--multiple .ant-select-selection__choice:hover {*/
-/*            background-color: #176db7;*/
-/*            color: #ffffff;*/
-
-/*            .ant-select-selection__choice__remove {*/
-/*                color: #ffffff;*/
-/*            }*/
-/*        }*/
-/*    }*/
-/*}*/
-
-/*.dropDownMenu:global(.ant-select-dropdown.ant-select-dropdown--multiple .ant-select-dropdown-menu-item-selected .ant-select-selected-icon),*/
-/*.dropDownMenu:global(.ant-select-dropdown.ant-select-dropdown--multiple .ant-select-dropdown-menu-item:hover .ant-select-selected-icon) {*/
-/*    display: none;*/
-/*}*/
-
-/*.dropDownMenu:global(.ant-select-dropdown.ant-select-dropdown--multiple .ant-select-dropdown-menu-item:hover) {*/
-/*    .emptyCheckbox {*/
-/*        border-color: #176db7;*/
-/*    }*/
-/*}*/
+.span{
+  margin-left: 8px;
+}
+:global(.ant-select-dropdown--multiple){
+  max-width: 255px;
+}
+:global(.ant-select-dropdown-menu-item-selected):not(:hover){
+  background-color: white;
+}
```

Comment:
These selectors aren't scoped under `MultipleSelectField` component and will affect all `<Select>` from ant design components on the page.